src/garage/np/algos/off_policy_rl_algorithm.py

```diff
@@ -94,7 +93,7 @@ def train(self, runner):
                 runner.step_path = runner.obtain_samples(runner.step_itr)
                 last_return = self.train_once(runner.step_itr,
                                               runner.step_path)
-                if cycle == 0 and self.evaluate:
```

Comment:
@ahtsan Is this the correct way to use `self.evaluate`?

Reply:
I think it's fine without `self.evaluate` here because it only evaluates performance. The cases that we should put a guard are those modifying the policy/Q function, like the one I commented in `dqn.py` where we shouldn't update qf during first `min_buffer_size` steps.